src/js/dataMapping/navigation/menuOptions.jsx

```diff
@@ -133,7 +136,8 @@ export const downloadOptions = [
         callToAction: 'Download Raw Files',
         shouldOpenNewTab: true,
         enabled: true,
-        internalDomain: true
+        internalDomain: true,
+        externalLink: true
```

Comment:
Adding externalLink here changes the functionality of the homepage download buttons.

Reply:
@jonhill13 I believe you, but how/where? Now I really wish we had some integration tests we could use to check....

Reply:
in `components/homepage/download/DownloadDetail.jsx`.

The code below is now triggering.
```
        if (this.props.externalLink) {
            link = (
                <button
                    className="download-detail__link"
                    onClick={this.redirect}>
                    {this.props.callToAction}
                </button>
            );
        }
```

This is the location on the homepage.
<img width="1128" alt="Screen Shot 2021-06-23 at 5 05 15 PM" src="https://user-images.githubusercontent.com/14301977/123167909-3ad80680-d445-11eb-8f66-bc6bb59cbf96.png">

Reply:
Thanks, good catch. updated

Reply:
This also changes the behavior of the Download Center sidebar -  The redirect modal is coming up for Agency Submission Files and Database Download. I think we need to handle the distinction between internal domain / external link from that sidebar as well.